WebAPI/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WebNetSample.Business.Abstract;
+
+namespace WebNetSample.WebAPI.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class CategoriesController : ControllerBase
+{
+    private readonly ICategoryService _categoryService;
+
+    public CategoriesController(ICategoryService categoryService)
+    {
+        _categoryService = categoryService;
+    }
+
+    [HttpGet("getall")]
```

Comment:
Right now most of the Web APIs are trying to follow [RESTful](https://www.techtarget.com/searchapparchitecture/definition/RESTful-API) standard. In this case URL is created incorrectly: you don't need any postfix since your controller route matches endpoint route (`/api/categories`). 
Also pay attention, please, that routes in RESTful apps should be in _kebab-case_ template.

(Same case for products)